samples/python/client_side.py

```diff
@@ -0,0 +1,15 @@
+import google.auth.transport.requests
+
+
+def get_id_token_from_service_account(svc_account_file: str, target_audience: str):
+    id_token_credentials = (
+        google.oauth2.service_account.IDTokenCredentials.from_service_account_file(
+            svc_account_file,
```

Comment:
This is the same file as `./sa-private-key.json` in Connecting_to_Settle.md? May be useful to make that clearer for the reader.

Reply:
:heavy_check_mark: